config/certmanager/certificate.yaml

```diff
@@ -1,14 +1,14 @@
 # The following manifests contain a self-signed issuer CR and a certificate CR.
 # More document can be found at https://docs.cert-manager.io
-apiVersion: cert-manager.io/v1alpha2
+apiVersion: cert-manager.io/v1
```

Comment:
Can you help find out which cert manager version introduces the v1 version? we need to update the doc with the minimum requirement.

Reply:
Since version 1.0.0
https://cert-manager.io/docs/release-notes/release-notes-1.0/

I will also test with 1.0 to be sure.